src/js/main.js

```diff
@@ -17,7 +17,14 @@
       templateUrl: 'partials/user-wishes.html'
     })//END OF USER-WISHES
     .when ('/results', {
-      templateUrl: 'partials/results.html'
+      templateUrl: 'partials/results.html',
+      controller: function($http, $scope){
+        $http.get('http://wishcastr-staging.herokuapp.com/products/search.json?query='+$scope.query)
```

Comment:
You can accomplish the same without explicitly concatenating these values yourself. Probably not as big an issue when you've only got one value like this, but good to practice. See [the docs for `$http.get`](https://docs.angularjs.org/api/ng/service/$http#get) and [using a `config` object with `$http`](https://docs.angularjs.org/api/ng/service/$http#usage). Specifically pay attention to the `params` key in that object.